src/traffic_cop/traffic_cop.cpp

```diff
@@ -409,10 +410,11 @@ TrafficCopResult TrafficCop::RunExecutableQuery(const common::ManagedPointer<net
                    "Not in a valid txn. This should have been caught before calling this function.");
   const auto query_type = portal->GetStatement()->GetQueryType();
   const auto physical_plan = portal->OptimizeResult()->GetPlanNode();
-  NOISEPAGE_ASSERT(query_type == network::QueryType::QUERY_SELECT || query_type == network::QueryType::QUERY_INSERT ||
-                       query_type == network::QueryType::QUERY_CREATE_INDEX ||
-                       query_type == network::QueryType::QUERY_UPDATE || query_type == network::QueryType::QUERY_DELETE,
-                   "CodegenAndRunPhysicalPlan called with invalid QueryType.");
+  NOISEPAGE_ASSERT(
+      query_type == network::QueryType::QUERY_SELECT || query_type == network::QueryType::QUERY_INSERT ||
```

Comment:
Can you make this one aligned with the other ones as well?

Reply:
I keep trying to fix this, but clang format keeps putting it back into this format. When I change it manually, then it fails check-format.

Reply:
Ah no worries then, just something that caught my eye.